### PR TITLE
624 convert qpid to amqp 1.0

### PR DESCRIPTION
**Pull request by bmbouter**
I wanted to see what the test runners said about the code I have. I had developed it against 3.0. All of the unit tests pass locally, but sometimes the PR test runners point out other issues.

I will rebase onto 4.0 soon and switch the target branch to 4.0 later. I don't plan to merge this to the 3.0 branch.

### CODE REVIEW

**Comment by auvipy** [comment]
nice work, looking forward for final review

File: kombu/transport/qpid.py [hunk 1]
```diff
@@ -96,41 +99,51 @@
...
 
 OBJECT_ALREADY_EXISTS_STRING = 'object already exists'
 
-VERSION = (1, 0, 0)
-__version__ = '.'.join(map(str, VERSION))
+PROTON_PERIOD = 0.25 # The waiting period between ProtonThread check-ins
 
-PY3 = sys.version_info[0] == 3
+
+StartConsumer = namedtuple('StartConsumer', ['queue'])
```
Comment (daviddavis) [inline kombu/transport/qpid.py]: I think a comment here would help to describe what these named tuples are for.

Comment (bmbouter) [reply kombu/transport/qpid.py]: Great idea. I just pushed a change with docstrings for each object type.

File: requirements/extras/qpid.txt [hunk 2]
```diff
@@ -1,2 +1,2 @@
-qpid-python>=0.26
-qpid-tools>=0.26
+python-qpid-proton
+qpid-tools
```
Comment (auvipy) [inline requirements/extras/qpid.txt:2]: i get this error while trying to install qpid-tools File "/tmp/pip-install-9104bo_x/qpid-python_a14df52a368846c0b0474e8c8647e96c/setup.py", line 42
        raise DistutilsFileError, \
                                ^
    SyntaxError: invalid syntax
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.

**Pull request closed without merging**